api/holiday_acres_api/serializers.py

```diff
@@ -48,8 +49,40 @@ def update(self, horse, data):
 
         # If new feed was provided, put that in by removing
         # the old instance and creating a new instance
+        response = HttpResponse()
         if "feed" in data.keys():
 
+            for feed in data["feed"]:
+                scoopable_feed = [
+                    "PELLETS",
+                    "HAY_PELLETS",
+                    "FIBREMAX",
+                    "ALFALFA",
+                    "CARB_SAFE",
+                ]
+                # make sure amount is positive value
+                if feed["amount"] < 0:
+                    raise Exception("Amount must be a positive value")
+                # make sure 1st/2nd cut are only used for hay cut
+                if feed["feed_type"] == "HAY_CUT" and feed["unit"] not in [
+                    "FIRST_CUT",
+                    "SECOND_CUT",
+                ]:
+                    raise Exception("Haycut must be 1st cut or 2nd cut")
+                # make sure scoopable feeds are measured in scoops and handfuls
+                if feed["feed_type"] in scoopable_feed and feed["unit"] in [
+                    "FIRST_CUT",
+                    "SECOND_CUT",
+                    "CUP",
+                ]:
+                    raise Exception("Must measure by scoop or handful")
+                # make sure oil is measured in cups
+                if feed["feed_type"] == "OIL" and feed["unit"] not in "CUP":
+                    raise Exception("Oil must be measured in Cups")
+                # default to 0 and scoop if feed_type = NONE
+                if feed["feed_type"] == "NONE":
+                    feed["amount"] = 0
```

Comment:
I'd actually like to leave this value blank. The model does allow the value to be null: https://github.com/mstephenhunt/holiday_acres/blob/main/api/holiday_acres_api/models/Feeds.py#L36

Reply:
Ok, I was thinking "NONE" was an option that could be selected. But that makes more sense.